op-acceptor/flags/flags.go

```diff
@@ -53,6 +54,12 @@ var (
 		EnvVars: opservice.PrefixEnvVar(EnvVarPrefix, "ALLOW_SKIPS"),
 		Usage:   "Allow tests to be skipped instead of failing when preconditions are not met.",
 	}
+	DefaultTimeout = &cli.DurationFlag{
```

Comment:
The same from above - the sorta standard across all frameworks is --timeout, i'd just name it that

Reply:
@scharissis covered well my thoughts https://github.com/ethereum-optimism/infra/pull/262#discussion_r2011030586